http/1.5/modules/ROOT/pages/http-consume-web-svc-task.adoc

```diff
@@ -1,33 +1,96 @@
-= To Consume a REST Service (Studio)
+= Consume a REST Service Example - Mule 4
 :page-aliases: connectors::http/http-consume-web-svc-task.adoc
 
-You can design an app to consume a REST service by setting up HTTP Listener and Request operations. In this procedure, you first consume a REST API service using primarily default HTTP configurations. Next, you test the HTTP request. You edit the configuration to make a secure HTTPS request. Finally, you test the HTTPS request.
+In the following example you configure Anypoint Connector for HTTP (HTTP Connector) to consume a REST service by setting up the HTTP *Listener* and *Request* operations.
 
-. From the Mule Palette, drag an HTTP Listener operation to a new flow.
-. Set up the Listener configuration: Click + and set the default host, port and base path. For example, host = 0.0.0.0, port = 8081, and no base path.
-. In General, set Path to */trigger*.
-. Accept the other default Listener settings, and save the configuration.
-. From Mule Palette, drag an HTTP Request operation to the flow.
-. Set up the Request configuration:
+To accomplish this example, you must: +
```

Comment:
I haven't been marking this, but I am not sure "accomplish" is the right word here. This might be something to ask Kim. Maybe use "implement" instead.